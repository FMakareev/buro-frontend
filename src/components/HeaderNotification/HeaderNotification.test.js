

test('HeaderNotification: пользователь не авторизован', async () => {
  /**
   * в редаксе нет пользователя, вернет null
   * */
});

test('HeaderNotification: Загрузка', async () => {
  /**
   * состояние загрузки
   * */
});

test('HeaderNotification: ошибка', async () => {
  /**
   * состояние ошибки
   * */
});

test('HeaderNotification: загрузилось', async () => {
  /**
   * состояние загрузка завершена
   * */
});
