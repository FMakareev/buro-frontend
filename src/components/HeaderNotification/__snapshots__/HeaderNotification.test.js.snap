// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNotification: Загрузка 1`] = `
<div
  className="sc-bxivhb dUiVDW"
/>
`;

exports[`HeaderNotification: загрузилось нотификации есть 1`] = `
<div
  className="sc-bxivhb dUiVDW"
>
  <button
    className="sc-1jxwge6-0 sc-18ch56c-0 sc-1gptvzz-1 kJgcoy"
    color="color1"
    fill="color1"
    fontSize="40px"
    onClick={[Function]}
  >
    <div
      className="sc-1jxwge6-0 sc-18ch56c-0 sc-1gptvzz-0 dvwLkW"
      color="color1"
    >
      2
    </div>
    <svg
      fill="inherit"
      height="1em"
      viewBox="0 0 40 40"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="20"
        cy="20"
        fill="white"
        r="20"
      />
      <path
        d="M20 0C8.95437 0 0 8.95431 0 20C0 31.0457 8.95437 40 20 40C31.0457 40 40 31.0457 40 20C40 8.95431 31.0457 0 20 0ZM20 38.75C9.66125 38.75 1.25 30.3388 1.25 20C1.25 9.66125 9.66125 1.25 20 1.25C30.3388 1.25 38.75 9.66125 38.75 20C38.75 30.3388 30.3388 38.75 20 38.75Z"
        fill="inherit"
      />
      <path
        d="M26.5056 22.6567L27.479 19.0238C28.2383 16.2587 26.8879 13.9279 25.5007 12.7721C25.1091 12.4464 24.6798 12.1673 24.2315 11.9221L24.2977 11.6749C24.5961 10.5613 23.9425 9.41508 22.8408 9.11987C21.7391 8.82466 20.6 9.49049 20.3016 10.6041L20.2252 10.8893C19.7633 10.8881 19.3023 10.9143 18.8497 10.9939C16.8817 11.3402 14.5826 12.8299 13.8827 15.3781L12.9086 19.0134C12.2172 21.5937 10.7225 23.566 8.46599 24.8758C8.33151 24.954 8.25105 25.1009 8.25717 25.2571C8.26339 25.4129 8.3558 25.5524 8.49633 25.6177L12.1117 27.296C12.7936 27.6125 13.4162 27.8863 14.0054 28.1316C14.3323 29.7007 15.4641 30.9546 16.9611 31.3558C18.4585 31.757 20.0662 31.2362 21.1336 30.0403C21.7636 30.1221 22.4364 30.1959 23.1815 30.2622L27.1512 30.6163C27.3055 30.6304 27.4549 30.5557 27.5387 30.4235C27.6223 30.2919 27.6262 30.124 27.5485 29.9889C26.1875 27.6194 25.8368 25.1525 26.5056 22.6567ZM21.1008 10.8183C21.2798 10.1502 21.9634 9.75061 22.6243 9.92772C23.2853 10.1048 23.6775 10.7926 23.4985 11.4607L23.4776 11.5387C22.7117 11.2114 21.8983 11.0004 21.0745 10.9164L21.1008 10.8183ZM16.1722 16.4245C16.1714 16.4242 16.1706 16.424 16.1698 16.4238C15.9496 16.3631 15.8205 16.1337 15.8814 15.911C16.4002 14.0231 18.208 13.2256 19.195 13.0523C20.9252 12.7475 22.8211 13.2568 24.178 14.3867C24.3538 14.5329 24.378 14.7961 24.232 14.9748C24.0858 15.1529 23.8248 15.1791 23.6488 15.0319C22.4748 14.0537 20.8304 13.6113 19.3333 13.8756C18.5391 14.0152 17.0869 14.6474 16.6798 16.1301C16.6185 16.3523 16.3916 16.4832 16.1722 16.4245ZM20.1035 29.8933C19.2734 30.5636 18.1903 30.8188 17.1773 30.5474C16.1646 30.2761 15.3547 29.5136 14.9707 28.5188L14.9711 28.5189C15.1234 28.5771 15.2717 28.6311 15.4201 28.6847C15.493 28.7112 15.5676 28.7398 15.6399 28.7657C15.6935 28.7848 15.7455 28.8018 15.7984 28.8198C16.9368 29.2166 18.0064 29.5037 19.1889 29.7309C19.2582 29.7443 19.3255 29.7584 19.3956 29.7716C19.483 29.7876 19.5741 29.8025 19.6634 29.8178C19.8037 29.8424 19.9431 29.8672 20.0876 29.8903C20.0933 29.8914 20.0982 29.8923 20.1035 29.8933Z"
        fill="inherit"
      />
    </svg>
  </button>
</div>
`;

exports[`HeaderNotification: загрузилось нотификаций нет 1`] = `
<div
  className="sc-bxivhb dUiVDW"
>
  <button
    className="sc-1jxwge6-0 sc-18ch56c-0 sc-1gptvzz-1 kJgcoy"
    color="color1"
    fill="color1"
    fontSize="40px"
    onClick={[Function]}
  >
    <svg
      fill="inherit"
      height="40"
      viewBox="0 0 40 40"
      width="40"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M26.5765 22.3973L27.5288 18.8433C28.272 16.1384 26.9108 13.8476 25.5192 12.7077C25.1206 12.3811 24.6954 12.0977 24.2516 11.8531L24.3124 11.6262C24.6043 10.5368 23.9454 9.41034 22.8436 9.11512C21.7419 8.8199 20.608 9.46602 20.3161 10.5554L20.2461 10.8166C19.7875 10.8161 19.3268 10.8524 18.8684 10.9294C16.9051 11.258 14.6173 12.705 13.9323 15.1976L12.9794 18.7539C12.303 21.2781 10.8212 23.2023 8.57499 24.4732C8.44114 24.5492 8.36159 24.6926 8.36855 24.8457C8.37551 24.9987 8.46865 25.1355 8.60933 25.2003L12.2287 26.8642C12.903 27.1741 13.5271 27.4472 14.1243 27.6945C14.4596 29.2327 15.5967 30.467 17.0929 30.8679C18.5896 31.269 20.192 30.7683 21.2513 29.6029C21.8896 29.6871 22.5634 29.7622 23.2982 29.8303L27.2642 30.1989C27.4183 30.2135 27.5671 30.1411 27.65 30.0121C27.7329 29.8836 27.7358 29.7192 27.6575 29.5864C26.2863 27.2577 25.9223 24.8389 26.5765 22.3973ZM21.1154 10.7696C21.2905 10.116 21.9709 9.7283 22.6319 9.90541C23.2929 10.0825 23.6883 10.7585 23.5131 11.4121L23.4939 11.484C22.7277 11.1576 21.9184 10.9458 21.0911 10.8601L21.1154 10.7696ZM17.3052 30.0774C16.2933 29.8062 15.481 29.0558 15.0918 28.0803C15.9606 28.4105 16.7819 28.6797 17.6152 28.903C18.4516 29.1271 19.3003 29.3054 20.2218 29.4536C19.3963 30.1045 18.3171 30.3485 17.3052 30.0774ZM23.3715 29.0148C22.5695 28.9401 21.8381 28.858 21.1515 28.7638C21.1507 28.7636 21.1499 28.7634 21.1487 28.7631C20.8459 28.7213 20.5528 28.6771 20.2668 28.6301C20.2644 28.6295 20.2623 28.6293 20.2599 28.6287C19.9978 28.5855 19.7429 28.5393 19.4913 28.4905C19.4441 28.4812 19.3968 28.472 19.3501 28.4624C19.1326 28.419 18.9183 28.3734 18.7059 28.325C18.6628 28.3151 18.619 28.3059 18.5759 28.2956C18.0743 28.179 17.5839 28.0476 17.0915 27.8983C17.0476 27.8848 17.0035 27.8705 16.9595 27.857C16.7552 27.7934 16.5497 27.7273 16.3428 27.657C16.2937 27.6405 16.2446 27.6239 16.1952 27.6069C15.9604 27.5258 15.7235 27.4411 15.4827 27.3503C15.4737 27.3471 15.4651 27.3439 15.4561 27.3407C15.1869 27.2393 14.9133 27.1317 14.6329 27.0176C14.6317 27.0173 14.6305 27.017 14.6293 27.0166C13.9851 26.7547 13.308 26.4585 12.5731 26.1209L9.65187 24.7783C11.7258 23.4053 13.1125 21.4543 13.7787 18.9681L14.7309 15.4142C15.32 13.2702 17.3051 12.0225 19.0106 11.7366C19.5177 11.6519 20.0277 11.6238 20.5334 11.6436C20.5376 11.6439 20.5418 11.6455 20.5461 11.6458C21.5898 11.6908 22.6403 11.9617 23.5825 12.4306C23.5874 12.4332 23.5932 12.4322 23.5982 12.4344C24.0905 12.6815 24.56 12.9818 24.9945 13.3375C26.2901 14.3991 27.3297 16.4454 26.7301 18.6272L25.7771 22.1835C25.1384 24.5674 25.4136 27.0171 26.5709 29.3126L23.3715 29.0148Z"
        fill="inherit"
      />
      <path
        d="M24.243 14.6063C22.825 13.4965 20.8653 12.9706 19.0926 13.2271C18.0817 13.3733 16.2389 14.0997 15.7443 15.8963C15.6862 16.1082 15.8232 16.3303 16.0512 16.3931C16.052 16.3933 16.0528 16.3935 16.0537 16.3937C16.2804 16.4545 16.5111 16.3337 16.5691 16.1222C16.9574 14.711 18.4378 14.1359 19.2512 14.018C20.785 13.7955 22.4847 14.2526 23.7117 15.2131C23.8957 15.3572 24.1634 15.3379 24.3102 15.1703C24.457 15.0024 24.4264 14.7498 24.243 14.6063Z"
        fill="inherit"
      />
      <path
        d="M20 0C8.95437 0 0 8.95431 0 20C0 31.0457 8.95437 40 20 40C31.0457 40 40 31.0457 40 20C40 8.95431 31.0457 0 20 0ZM20 38.75C9.66125 38.75 1.25 30.3388 1.25 20C1.25 9.66125 9.66125 1.25 20 1.25C30.3388 1.25 38.75 9.66125 38.75 20C38.75 30.3388 30.3388 38.75 20 38.75Z"
        fill="inherit"
      />
    </svg>
  </button>
</div>
`;

exports[`HeaderNotification: ошибка 1`] = `
<div
  className="sc-bxivhb dUiVDW"
/>
`;

exports[`HeaderNotification: пользователь не авторизован 1`] = `
<div
  className="sc-bxivhb dUiVDW"
/>
`;
